### PR TITLE
Move old "Whats New in Mixxx" pages into blog

### PR DESCRIPTION
**Pull request by Holzhaus**

### CODE REVIEW

File: pages/news/2013-02-18-whats-new-in-mixxx-1-11.html [hunk 1]
```diff
@@ -1,12 +1,12 @@
-{% extends "base.html" %}
+title: What's New in Mixxx 1.11
+author: Albert
```
Comment (Be-ing) [inline pages/news/2013-02-18-whats-new-in-mixxx-1-11.html:2]:
```suggestion
author: ywwg
```

Comment (Holzhaus) [reply pages/news/2013-02-18-whats-new-in-mixxx-1-11.html:2]: Are you sure?

```
commit e049c5f6d6c5bd4da66fafbc7f0d73c488c8967d
Author: Albert Santoni <alberts@mixxx.org>
Date:   Mon Feb 18 01:14:33 2013 +0100

    Updated for Mixxx 1.11, added What's New in 1.11 page
```

Comment (Be-ing) [reply pages/news/2013-02-18-whats-new-in-mixxx-1-11.html:2]: Oh sorry, I got confused. I didn't see you moved the pages for both 1.11 and 2.0. Albert wrote the 1.11 one, ywwg wrote the 2.0 one.

**Comment by ronso0** [comment]
btw: can we transition to a real blog-like view, like showing the 2-4 most recent posts on first page and allow to browse through older news on conscutive pages? and maybe link to the complete 'Older News' overview ?

**Comment by ronso0** [comment]
Or just have `[prev. post]` and `[next post]` links also above each blog post to allow browsing news without having to scroll all the way down to the bottom of the page?

**Comment by Holzhaus** [comment]
Merge?

**Pull request merged**